src/client.js

```diff
@@ -794,6 +798,16 @@ function SugarApi(args) {
             _bulkQueues = {};
         },
 
+        /**
+         * Clear the request call queue.
+         *
+         * @memberOf Api
+         * @instance
+         */
+        clearRequestQueue: function() {
+            _rqueue = [];
+        },
```

Comment:
This should not be a public function. If the request queue needs to be cleared, it is Ventana's responsibility to know. The request queue is entirely an internal mechanism.

Reply:
I retested code and this method isn't needed now.